scripts/quests/windurst/Something_Fishy.lua

```diff
@@ -0,0 +1,140 @@
+-----------------------------------
+-- Something Fishy
+-----------------------------------
+-- Log ID: 2, Quest ID: 52
+-- Tokaka : !pos 240 -189.678 -2 66.335
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.WINDURST, xi.quest.id.windurst.SOMETHING_FISHY)
+
+quest.reward =
+{
+    fame     = 60,
+    fameArea = xi.quest.fame_area.WINDURST,
+    gil      = 70,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE
+        end,
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Tokaka'] = quest:progressEvent(208, 0, xi.items.BASTORE_SARDINE),
+
+            onEventFinish =
+            {
+                [208] = function(player, csid, option, npc)
+                    quest:begin(player)
+                    quest:setVar(player, 'Dialog', 1)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.PORT_WINDURST] =
+        {
+            ['Tokaka'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:event(209, 0, xi.items.BASTORE_SARDINE)
+                end,
+
+                onTrade = function(player, npc, trade)
+                    if
+                        npcUtil.tradeHasExactly(trade, xi.items.BASTORE_SARDINE) and
+                        quest:getVar(player, 'Dialog') == 1
+                    then
+                        return quest:event(210, 70, xi.items.BASTORE_SARDINE)
+                    end
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status ~= QUEST_AVAILABLE
+        end,
```

Comment:
Isnt this dead code? Let me explain...
Quest can only be available, acceptd and completed and we already have specific sections for all 3

Reply:
Yea I realized that they are not different. I will combine the two in the morning and squash.

Reply:
Got it - little rusty been a few weeks since Ive done this. Thanks for the comments to make it right.